python/mxnet/log.py

```diff
@@ -63,12 +63,38 @@ def format(self, record):
 def getLogger(name=None, filename=None, filemode=None, level=WARNING):
     """Get customized logger.
 
-    Args:
-        name: Name of the logger.
-        level: Level to log.
+    Parameters
+    ----------
+    name: str, optional
+        Name of the logger
```

Comment:
Minor: The body paragraph for all the parameters and return objects should end in a '.' (period)